program/src/instructions/compressed/take_bid.rs

```diff
@@ -94,6 +102,7 @@ impl<'info> Validate<'info> for TakeBidCompressed<'info> {
             TcompError::BrokerMismatch
         );
 
+        // TODO: why are we checking here manually instead of using the validate_cosigner helper fn? Inconsistent.
```

Comment:
Existing handlers have to be backwards compatible, new handlers can do it the right way from the beginning.